java-api/src/main/java/com/sap/cloud/security/x509/X509Constants.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * SPDX-FileCopyrightText: 2018-2021 SAP SE or an SAP affiliate company and Cloud Security Client Java contributors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.sap.cloud.security.x509;
+
+public final class X509Constants {
+
+	private X509Constants() {
+	}
+
+	public static final String FWD_CLIENT_CERT_HEADER = "x-forwarded-client-cert";
```

Comment:
as it is only used in X509CertificateExtractor.java and in tests I would move the constant to that class.

Reply:
yes for now, but with the next validators there will be sys env variables that will control the validator usage, I just removed it to make this PR slim

Reply:
ok